frontend/src/metabase/components/UserAvatar/UserAvatar.tsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable react/prop-types */
+import React from "react";
+
+import MetabaseUtils from "metabase/lib/utils";
+import { Avatar, AvatarProps } from "./UserAvatar.styled";
+
+interface UserAvatarProps extends AvatarProps {
+  user: User;
+}
+
+interface GroupProps {
+  user: Group;
+}
+
+interface User {
+  first_name: string | null;
+  last_name: string | null;
+  common_name: string;
+  email?: string;
+}
+
+interface Group {
```

Comment:
Should we combine `Group` and `User`? I see that `User` already has `email?: string`

Reply:
I want to make them separate so we know we pass 2 kinds of data. One is typical users, while the other one is just a hack for displaying group avatars.

https://github.com/metabase/metabase/blob/904a8e467f72d8faaef1abbe0765c446f8ba4be2/frontend/src/metabase/admin/people/components/GroupsListing.jsx#L184

Does this sound reasonable?